Updating interview process (#26)
sourcegraph/careers: job-descriptions/software-engineer.md

1.  You [apply here](https://hire.withgoogle.com/public/jobs/sourcegraphcom/view/P_AAAAAADAAADP_pY7jAAAXU)!
1.  We schedule a 30 minute call to chat with you about Sourcegraph and to find out what you are looking for in your next role.

> 1.  We give you a well defined problem to solve and a 2 hour time limit.

Reviewer: I thought of a better wording. What do you think?

```suggestion
1.  We give you a take home coding assignment which won't take you more than 2 hours to finish.
```

Author: Looks good, I think that's more clear.

> 1.  We gather a code sample. We'll give you a well defined problem to solve and a 2 hour time limit.

Reviewer: We got feedback that "code sample" sounded clinical (e.g. https://sourcegraph.slack.com/archives/C168VP0FM/p1550018256006800), so I would delete this bit.

```suggestion
1.  We give you a well defined problem to solve and a 2 hour time limit.
```
